default.build

```diff
@@ -50,6 +50,8 @@
 			<arg value="msbuild" />
 			<arg value="/t:Restore" />
 			<arg value="${root.dir}/src/NHibernate.sln" />
+			<arg value="/p:Platform=&quot;Any CPU&quot;" />
+			<arg value="/p:Configuration=&quot;${build.config}&quot;"/>
```

Comment:
Why these changes?

Reply:
Check you environment variables. You likely have a configuration or platform one defined in the command line you use for generating async code. They must be unset to avoid having their values messing with MsBuild parameters.

Reply:
After a reboot it worked again without the specific platform and configuration switches.  However given that these are explicitly specified for the build, is it not also worthwhile doing so for the solution-restore step?  Would avoid similar unexpected issues in future.

Reply:
Granted. Still a bit unrelated to this PR and commit.

Reply:
That's true